internal/db/schema.go

```diff
@@ -0,0 +1,44 @@
+package db
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Schema struct {
```

Comment:
rewrite these to use Loaders and do some kind of union query? it's fine if union is just a loop for now